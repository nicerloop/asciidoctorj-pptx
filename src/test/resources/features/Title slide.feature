Feature: Title slide

    Background:
        Given I have an empty temporary asciidoc document

    Scenario: Title
        Given the asciidoc document has a title
        When I convert the asciidoc document to an Office Open XML slideshow
        Then the slideshow has 1 slide
        And the slide looks like expected

    Scenario: Title and subtitle
        Given the asciidoc document has a title and a subtitle
        When I convert the asciidoc document to an Office Open XML slideshow
        Then the slideshow has 1 slide
        And the slide looks like expected

    # Scenario: Title and author
    #     Given the asciidoc document has a title
    #     And the asciidoc document has an author
    #     When I convert the asciidoc document to an Office Open XML slideshow
    #     Then the slideshow has 1 slide
    #     And the slide looks like expected

    # Scenario: Title and date
    #     Given the asciidoc document has a title
    #     And the asciidoc document has a version and a date
    #     When I convert the asciidoc document to an Office Open XML slideshow
    #     Then the slideshow has 1 slide
    #     And the slide looks like expected

    # Scenario: Title, subtitle, author and date
    #     Given the asciidoc document has a title and a subtitle
    #     And the asciidoc document has an author and a date
    #     When I convert the asciidoc document to an Office Open XML slideshow
    #     Then the slideshow has 1 slide
    #     And the slide looks like expected
