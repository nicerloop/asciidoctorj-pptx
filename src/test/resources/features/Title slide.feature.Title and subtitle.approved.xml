<?xml version="1.0" encoding="UTF-8"?><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" contentScriptType="text/ecmascript" contentStyleType="text/css" preserveAspectRatio="xMidYMid meet" style="color-interpolation:auto; color-rendering:auto; fill:black; fill-opacity:1; font-family:'Dialog'; font-size:12px; font-style:normal; font-weight:normal; image-rendering:auto; shape-rendering:auto; stroke:black; stroke-dasharray:none; stroke-dashoffset:0; stroke-linecap:square; stroke-linejoin:miter; stroke-miterlimit:10; stroke-opacity:1; stroke-width:1; text-rendering:auto;" version="1.0" zoomAndPan="magnify">
  <defs id="genericDefs"/>
  <g>
    <g style="color-interpolation:linearRGB; color-rendering:optimizeQuality; fill:white; image-rendering:optimizeQuality; stroke:white; text-rendering:optimizeLegibility;">
      <rect height="540" style="stroke:none;" width="720" x="0" y="0"/>
    </g>
    <g style="color-interpolation:linearRGB; color-rendering:optimizeQuality; fill:rgb(255,255,255); fill-opacity:0; image-rendering:optimizeQuality; stroke:rgb(255,255,255); stroke-opacity:0; text-rendering:optimizeLegibility;">
      <rect height="540" style="stroke:none;" width="720" x="0" y="0"/>
      <rect height="540" style="stroke:none;" width="720" x="0" y="0"/>
      <rect height="540" style="stroke:none;" width="720" x="0" y="0"/>
      <path d="M54 167.75 L666 167.75 L666 283.5 L54 283.5 Z" style="stroke:none; stroke-linecap:butt; stroke-linejoin:round; stroke-width:0.25;"/>
    </g>
    <g style="color-interpolation:linearRGB; color-rendering:optimizeQuality; image-rendering:optimizeQuality; stroke-linecap:butt; stroke-linejoin:round; stroke-width:0.25; text-rendering:optimizeLegibility;">
      <path d="M341.9912 210.4062 Q341.9912 210.8281 341.9443 211.125 Q341.8975 211.4219 341.8037 211.6094 Q341.71 211.7812 341.5693 211.875 Q341.4287 211.9531 341.2568 211.9531 L333.4287 211.9531 L333.4287 236.0625 Q333.4287 236.2344 333.335 236.375 Q333.2568 236.5 333.0381 236.5781 Q332.835 236.6406 332.4756 236.6875 Q332.1318 236.75 331.585 236.75 Q331.0693 236.75 330.6943 236.6875 Q330.335 236.6406 330.1162 236.5781 Q329.9131 236.5 329.8193 236.375 Q329.7412 236.2344 329.7412 236.0625 L329.7412 211.9531 L321.9131 211.9531 Q321.7412 211.9531 321.6006 211.875 Q321.4756 211.7812 321.3818 211.6094 Q321.3037 211.4219 321.2412 211.125 Q321.1943 210.8281 321.1943 210.4062 Q321.1943 210 321.2412 209.6875 Q321.3037 209.375 321.3818 209.1875 Q321.4756 209 321.6006 208.9219 Q321.7412 208.8281 321.9131 208.8281 L341.2568 208.8281 Q341.4287 208.8281 341.5693 208.9219 Q341.71 209 341.8037 209.1875 Q341.8975 209.375 341.9443 209.6875 Q341.9912 210 341.9912 210.4062 ZM349.1357 236.0938 Q349.1357 236.2656 349.042 236.3906 Q348.9639 236.5 348.7764 236.5938 Q348.5889 236.6719 348.2451 236.7031 Q347.9014 236.75 347.3545 236.75 Q346.8389 236.75 346.4951 236.7031 Q346.1514 236.6719 345.9482 236.5938 Q345.7451 236.5 345.667 236.3906 Q345.5889 236.2656 345.5889 236.0938 L345.5889 216.75 Q345.5889 216.5938 345.667 216.4688 Q345.7451 216.3438 345.9482 216.2656 Q346.1514 216.1719 346.4951 216.125 Q346.8389 216.0781 347.3545 216.0781 Q347.9014 216.0781 348.2451 216.125 Q348.5889 216.1719 348.7764 216.2656 Q348.9639 216.3438 349.042 216.4688 Q349.1357 216.5938 349.1357 216.75 L349.1357 236.0938 ZM349.542 210.2188 Q349.542 211.4688 349.0732 211.9219 Q348.6045 212.375 347.3389 212.375 Q346.0889 212.375 345.6201 211.9375 Q345.167 211.4844 345.167 210.2656 Q345.167 209.0156 345.6357 208.5625 Q346.1045 208.1094 347.3857 208.1094 Q348.6201 208.1094 349.0732 208.5625 Q349.542 209 349.542 210.2188 ZM365.749 234.6719 Q365.749 235.2969 365.6553 235.6562 Q365.5771 236.0156 365.4053 236.1875 Q365.2334 236.3594 364.8896 236.5156 Q364.5459 236.6719 364.1084 236.7656 Q363.6709 236.8594 363.1709 236.9219 Q362.6709 236.9844 362.1865 236.9844 Q360.6709 236.9844 359.5928 236.5938 Q358.5303 236.1875 357.8428 235.3906 Q357.1553 234.5781 356.8428 233.3438 Q356.5303 232.1094 356.5303 230.4375 L356.5303 219.1406 L353.8271 219.1406 Q353.499 219.1406 353.2959 218.7969 Q353.1084 218.4531 353.1084 217.6719 Q353.1084 217.2656 353.1553 216.9844 Q353.2178 216.7031 353.2959 216.5312 Q353.3896 216.3438 353.5303 216.2656 Q353.6709 216.1875 353.8428 216.1875 L356.5303 216.1875 L356.5303 211.5938 Q356.5303 211.4375 356.6084 211.3125 Q356.6865 211.1875 356.8896 211.0938 Q357.0928 211 357.4365 210.9531 Q357.7803 210.9062 358.2959 210.9062 Q358.8271 210.9062 359.1709 210.9531 Q359.5146 211 359.7021 211.0938 Q359.9053 211.1875 359.9834 211.3125 Q360.0771 211.4375 360.0771 211.5938 L360.0771 216.1875 L365.0459 216.1875 Q365.2178 216.1875 365.3428 216.2656 Q365.4678 216.3438 365.5615 216.5312 Q365.6553 216.7031 365.7021 216.9844 Q365.749 217.2656 365.749 217.6719 Q365.749 218.4531 365.5459 218.7969 Q365.3584 219.1406 365.0459 219.1406 L360.0771 219.1406 L360.0771 229.9219 Q360.0771 231.9219 360.6709 232.9375 Q361.2646 233.9531 362.7803 233.9531 Q363.2803 233.9531 363.6553 233.8594 Q364.0459 233.7656 364.3428 233.6562 Q364.6553 233.5469 364.8584 233.4531 Q365.0771 233.3594 365.249 233.3594 Q365.3584 233.3594 365.4521 233.4219 Q365.5615 233.4688 365.6084 233.625 Q365.6553 233.7656 365.7021 234.0312 Q365.749 234.2812 365.749 234.6719 ZM373.9717 236.0938 Q373.9717 236.2656 373.8779 236.3906 Q373.7998 236.5 373.6123 236.5938 Q373.4248 236.6719 373.0811 236.7031 Q372.7373 236.75 372.1904 236.75 Q371.6748 236.75 371.3311 236.7031 Q370.9873 236.6719 370.7842 236.5938 Q370.5811 236.5 370.5029 236.3906 Q370.4248 236.2656 370.4248 236.0938 L370.4248 207.3906 Q370.4248 207.2188 370.5029 207.0938 Q370.5811 206.9531 370.7842 206.875 Q370.9873 206.7812 371.3311 206.75 Q371.6748 206.7031 372.1904 206.7031 Q372.7373 206.7031 373.0811 206.75 Q373.4248 206.7812 373.6123 206.875 Q373.7998 206.9531 373.8779 207.0938 Q373.9717 207.2188 373.9717 207.3906 L373.9717 236.0938 ZM397.0225 225.5781 Q397.0225 226.4219 396.6006 226.7812 Q396.1943 227.125 395.6475 227.125 L382.9756 227.125 Q382.9756 228.7344 383.2881 230.0312 Q383.6162 231.3125 384.3662 232.25 Q385.1318 233.1719 386.335 233.6719 Q387.5381 234.1562 389.2725 234.1562 Q390.6475 234.1562 391.7256 233.9375 Q392.8037 233.7031 393.585 233.4219 Q394.3662 233.1406 394.8662 232.9219 Q395.3818 232.6875 395.6318 232.6875 Q395.7881 232.6875 395.8975 232.7656 Q396.0225 232.8438 396.085 233 Q396.1475 233.1406 396.1787 233.4219 Q396.21 233.6875 396.21 234.0625 Q396.21 234.3438 396.1787 234.5469 Q396.1631 234.75 396.1318 234.9219 Q396.1006 235.0781 396.0225 235.2031 Q395.96 235.3281 395.835 235.4531 Q395.7256 235.5781 395.1318 235.8438 Q394.5537 236.1094 393.6318 236.375 Q392.71 236.625 391.4912 236.8281 Q390.2881 237.0312 388.9131 237.0312 Q386.5225 237.0312 384.7256 236.375 Q382.9287 235.7031 381.71 234.3906 Q380.4912 233.0781 379.8662 231.1094 Q379.2412 229.125 379.2412 226.5 Q379.2412 224.0156 379.8818 222.0312 Q380.5225 220.0312 381.7412 218.6562 Q382.96 217.2656 384.6787 216.5312 Q386.3975 215.7812 388.5225 215.7812 Q390.8037 215.7812 392.3975 216.5156 Q394.0068 217.25 395.0381 218.4844 Q396.0693 219.7188 396.5381 221.3906 Q397.0225 223.0469 397.0225 224.9375 L397.0225 225.5781 ZM393.46 224.5312 Q393.5225 221.7344 392.2256 220.1562 Q390.9287 218.5625 388.3662 218.5625 Q387.0537 218.5625 386.0693 219.0625 Q385.085 219.5469 384.4131 220.3594 Q383.7568 221.1719 383.3818 222.2656 Q383.0225 223.3438 382.9756 224.5312 L393.46 224.5312 Z" style="stroke:none;"/>
    </g>
    <g style="color-interpolation:linearRGB; color-rendering:optimizeQuality; fill:rgb(255,255,255); fill-opacity:0; image-rendering:optimizeQuality; stroke:rgb(255,255,255); stroke-linecap:butt; stroke-linejoin:round; stroke-opacity:0; stroke-width:0.25; text-rendering:optimizeLegibility;">
      <path d="M54 167.75 L666 167.75 L666 283.5 L54 283.5 Z" style="fill:none;"/>
    </g>
    <g style="color-interpolation:linearRGB; color-rendering:optimizeQuality; fill:rgb(255,255,255); fill-opacity:0; image-rendering:optimizeQuality; stroke:rgb(255,255,255); stroke-linecap:butt; stroke-linejoin:round; stroke-opacity:0; stroke-width:0.25; text-rendering:optimizeLegibility;">
      <path d="M108 306 L612 306 L612 444 L108 444 Z" style="stroke:none;"/>
    </g>
    <g style="color-interpolation:linearRGB; color-rendering:optimizeQuality; fill:rgb(137,137,137); image-rendering:optimizeQuality; stroke:rgb(137,137,137); stroke-linecap:butt; stroke-linejoin:round; stroke-width:0.25; text-rendering:optimizeLegibility;">
      <path d="M323.3594 334.9906 Q323.3594 336.4125 322.8281 337.5219 Q322.3125 338.6313 321.375 339.4125 Q320.4531 340.1781 319.1875 340.5688 Q317.9375 340.9594 316.4844 340.9594 Q315.4688 340.9594 314.5938 340.7875 Q313.7344 340.6156 313.0469 340.3656 Q312.375 340.1156 311.9062 339.85 Q311.4531 339.5844 311.2656 339.3969 Q311.0938 339.2094 311 338.9281 Q310.9219 338.6313 310.9219 338.1469 Q310.9219 337.8031 310.9531 337.5844 Q310.9844 337.35 311.0469 337.2094 Q311.1094 337.0688 311.2031 337.0219 Q311.2969 336.9594 311.4219 336.9594 Q311.6406 336.9594 312.0312 337.225 Q312.4375 337.4906 313.0625 337.8031 Q313.6875 338.1156 314.5625 338.3969 Q315.4531 338.6625 316.6094 338.6625 Q317.4844 338.6625 318.2031 338.4281 Q318.9375 338.1938 319.4531 337.7719 Q319.9844 337.3344 320.2656 336.7094 Q320.5469 336.0844 320.5469 335.2875 Q320.5469 334.4281 320.1562 333.8188 Q319.7656 333.2094 319.125 332.7563 Q318.4844 332.2875 317.6562 331.9125 Q316.8438 331.5219 315.9844 331.1313 Q315.125 330.725 314.3125 330.2406 Q313.5 329.7563 312.8594 329.1 Q312.2188 328.4438 311.8125 327.5688 Q311.4219 326.6781 311.4219 325.4438 Q311.4219 324.1781 311.875 323.1938 Q312.3438 322.1938 313.1562 321.5219 Q313.9844 320.85 315.1094 320.5063 Q316.25 320.1469 317.5625 320.1469 Q318.2344 320.1469 318.9062 320.2719 Q319.5938 320.3813 320.1875 320.5844 Q320.7969 320.7719 321.2656 321.0219 Q321.7344 321.2563 321.875 321.4125 Q322.0312 321.5531 322.0781 321.6469 Q322.125 321.725 322.1562 321.8656 Q322.1875 321.9906 322.2031 322.1781 Q322.2188 322.3656 322.2188 322.6625 Q322.2188 322.9438 322.1875 323.1625 Q322.1719 323.3813 322.125 323.5375 Q322.0781 323.6781 321.9844 323.7563 Q321.9062 323.8188 321.7969 323.8188 Q321.625 323.8188 321.25 323.6 Q320.8906 323.3813 320.3594 323.1156 Q319.8281 322.8344 319.0938 322.6156 Q318.375 322.3813 317.4688 322.3813 Q316.625 322.3813 316 322.6156 Q315.375 322.8344 314.9688 323.2094 Q314.5625 323.5844 314.3594 324.1 Q314.1562 324.6156 314.1562 325.1938 Q314.1562 326.0375 314.5469 326.6469 Q314.9375 327.2563 315.5781 327.725 Q316.2344 328.1938 317.0625 328.5844 Q317.8906 328.975 318.75 329.3813 Q319.6094 329.7719 320.4375 330.2563 Q321.2656 330.725 321.9062 331.3813 Q322.5625 332.0219 322.9531 332.9125 Q323.3594 333.7875 323.3594 334.9906 ZM338.9531 340.2719 Q338.9531 340.3969 338.8906 340.4906 Q338.8438 340.5688 338.7031 340.6313 Q338.5625 340.6938 338.3281 340.725 Q338.1094 340.7563 337.7812 340.7563 Q337.4219 340.7563 337.1875 340.725 Q336.9688 340.6938 336.8281 340.6313 Q336.7031 340.5688 336.6562 340.4906 Q336.6094 340.3969 336.6094 340.2719 L336.6094 338.4125 Q335.4062 339.7406 334.2344 340.35 Q333.0625 340.9594 331.8594 340.9594 Q330.4531 340.9594 329.4844 340.4906 Q328.5312 340.0219 327.9375 339.225 Q327.3438 338.4125 327.0781 337.35 Q326.8281 336.2719 326.8281 334.7406 L326.8281 326.2094 Q326.8281 326.0844 326.875 326.0063 Q326.9375 325.9125 327.0938 325.85 Q327.25 325.7719 327.5 325.7563 Q327.75 325.725 328.125 325.725 Q328.5 325.725 328.75 325.7563 Q329 325.7719 329.1406 325.85 Q329.2969 325.9125 329.3594 326.0063 Q329.4219 326.0844 329.4219 326.2094 L329.4219 334.3969 Q329.4219 335.6313 329.5938 336.3813 Q329.7812 337.1156 330.1406 337.6469 Q330.5156 338.1625 331.0781 338.4594 Q331.6406 338.7406 332.3906 338.7406 Q333.3594 338.7406 334.3125 338.0531 Q335.2812 337.3656 336.3594 336.0375 L336.3594 326.2094 Q336.3594 326.0844 336.4062 326.0063 Q336.4688 325.9125 336.625 325.85 Q336.7812 325.7719 337.0156 325.7563 Q337.2656 325.725 337.6562 325.725 Q338.0312 325.725 338.2812 325.7563 Q338.5312 325.7719 338.6719 325.85 Q338.8125 325.9125 338.875 326.0063 Q338.9531 326.0844 338.9531 326.2094 L338.9531 340.2719 ZM356.6562 333.0531 Q356.6562 334.8813 356.25 336.35 Q355.8594 337.8031 355.0781 338.8344 Q354.3125 339.8656 353.1875 340.4125 Q352.0625 340.9594 350.6094 340.9594 Q349.9375 340.9594 349.3594 340.8188 Q348.7969 340.6938 348.25 340.3969 Q347.7031 340.1 347.1562 339.6469 Q346.6094 339.1938 346 338.5531 L346 340.2719 Q346 340.3969 345.9375 340.4906 Q345.875 340.5844 345.7344 340.6469 Q345.5938 340.6938 345.375 340.725 Q345.1719 340.7563 344.8438 340.7563 Q344.5312 340.7563 344.3125 340.725 Q344.0938 340.6938 343.9531 340.6469 Q343.8125 340.5844 343.7656 340.4906 Q343.7188 340.3969 343.7188 340.2719 L343.7188 319.3969 Q343.7188 319.2719 343.7656 319.1781 Q343.8281 319.0844 343.9688 319.0219 Q344.125 318.9594 344.375 318.9281 Q344.625 318.8969 345 318.8969 Q345.3906 318.8969 345.6406 318.9281 Q345.8906 318.9594 346.0312 319.0219 Q346.1719 319.0844 346.2344 319.1781 Q346.2969 319.2719 346.2969 319.3969 L346.2969 327.8188 Q346.9219 327.1781 347.5 326.7406 Q348.0938 326.3031 348.6562 326.0375 Q349.2188 325.7563 349.7812 325.6313 Q350.3438 325.5063 350.9688 325.5063 Q352.5 325.5063 353.5781 326.1156 Q354.6719 326.725 355.3438 327.7563 Q356.0312 328.7719 356.3438 330.1469 Q356.6562 331.5219 356.6562 333.0531 ZM353.9375 333.35 Q353.9375 332.2719 353.7656 331.2563 Q353.6094 330.2406 353.2031 329.4594 Q352.7969 328.6781 352.125 328.2094 Q351.4531 327.725 350.4531 327.725 Q349.9531 327.725 349.4688 327.8656 Q348.9844 328.0063 348.4844 328.3344 Q347.9844 328.6625 347.4375 329.1781 Q346.9062 329.6938 346.2969 330.475 L346.2969 336.0844 Q347.3594 337.3813 348.3281 338.0688 Q349.2969 338.7406 350.3438 338.7406 Q351.3125 338.7406 352 338.2719 Q352.6875 337.8031 353.1094 337.0375 Q353.5469 336.2563 353.7344 335.3031 Q353.9375 334.3344 353.9375 333.35 ZM367.8438 339.2406 Q367.8438 339.6938 367.7812 339.9594 Q367.7188 340.225 367.5938 340.35 Q367.4688 340.475 367.2188 340.5844 Q366.9688 340.6938 366.6406 340.7563 Q366.3281 340.8344 365.9688 340.8813 Q365.6094 340.9281 365.25 340.9281 Q364.1562 340.9281 363.375 340.6469 Q362.5938 340.35 362.0938 339.7719 Q361.5938 339.1781 361.3594 338.2875 Q361.1406 337.3813 361.1406 336.1625 L361.1406 327.9438 L359.1719 327.9438 Q358.9375 327.9438 358.7969 327.6938 Q358.6562 327.4438 358.6562 326.8813 Q358.6562 326.5844 358.6875 326.3813 Q358.7344 326.1781 358.7969 326.0531 Q358.8594 325.9125 358.9531 325.8656 Q359.0625 325.8031 359.1875 325.8031 L361.1406 325.8031 L361.1406 322.4594 Q361.1406 322.35 361.1875 322.2563 Q361.25 322.1625 361.3906 322.1 Q361.5469 322.0219 361.7969 321.9906 Q362.0469 321.9594 362.4219 321.9594 Q362.8125 321.9594 363.0625 321.9906 Q363.3125 322.0219 363.4531 322.1 Q363.5938 322.1625 363.6562 322.2563 Q363.7188 322.35 363.7188 322.4594 L363.7188 325.8031 L367.3281 325.8031 Q367.4531 325.8031 367.5469 325.8656 Q367.6406 325.9125 367.7031 326.0531 Q367.7812 326.1781 367.8125 326.3813 Q367.8438 326.5844 367.8438 326.8813 Q367.8438 327.4438 367.7031 327.6938 Q367.5625 327.9438 367.3281 327.9438 L363.7188 327.9438 L363.7188 335.7875 Q363.7188 337.2406 364.1406 337.9906 Q364.5781 338.725 365.6875 338.725 Q366.0469 338.725 366.3281 338.6625 Q366.6094 338.5844 366.8281 338.5063 Q367.0469 338.4281 367.2031 338.3656 Q367.3594 338.2875 367.4844 338.2875 Q367.5625 338.2875 367.625 338.3344 Q367.7031 338.3656 367.7344 338.475 Q367.7812 338.5844 367.8125 338.7719 Q367.8438 338.9594 367.8438 339.2406 ZM373.8281 340.2719 Q373.8281 340.3969 373.7656 340.4906 Q373.7031 340.5688 373.5625 340.6313 Q373.4219 340.6938 373.1719 340.725 Q372.9219 340.7563 372.5312 340.7563 Q372.1562 340.7563 371.9062 340.725 Q371.6562 340.6938 371.5 340.6313 Q371.3594 340.5688 371.2969 340.4906 Q371.25 340.3969 371.25 340.2719 L371.25 326.2094 Q371.25 326.1 371.2969 326.0063 Q371.3594 325.9125 371.5 325.85 Q371.6562 325.7875 371.9062 325.7563 Q372.1562 325.725 372.5312 325.725 Q372.9219 325.725 373.1719 325.7563 Q373.4219 325.7875 373.5625 325.85 Q373.7031 325.9125 373.7656 326.0063 Q373.8281 326.1 373.8281 326.2094 L373.8281 340.2719 ZM374.125 321.4594 Q374.125 322.3656 373.7812 322.6938 Q373.4375 323.0219 372.5156 323.0219 Q371.6094 323.0219 371.2656 322.7094 Q370.9375 322.3813 370.9375 321.4906 Q370.9375 320.5844 371.2812 320.2563 Q371.625 319.9281 372.5469 319.9281 Q373.4531 319.9281 373.7812 320.2563 Q374.125 320.5688 374.125 321.4594 ZM385.9062 339.2406 Q385.9062 339.6938 385.8438 339.9594 Q385.7812 340.225 385.6562 340.35 Q385.5312 340.475 385.2812 340.5844 Q385.0312 340.6938 384.7031 340.7563 Q384.3906 340.8344 384.0312 340.8813 Q383.6719 340.9281 383.3125 340.9281 Q382.2188 340.9281 381.4375 340.6469 Q380.6562 340.35 380.1562 339.7719 Q379.6562 339.1781 379.4219 338.2875 Q379.2031 337.3813 379.2031 336.1625 L379.2031 327.9438 L377.2344 327.9438 Q377 327.9438 376.8594 327.6938 Q376.7188 327.4438 376.7188 326.8813 Q376.7188 326.5844 376.75 326.3813 Q376.7969 326.1781 376.8594 326.0531 Q376.9219 325.9125 377.0156 325.8656 Q377.125 325.8031 377.25 325.8031 L379.2031 325.8031 L379.2031 322.4594 Q379.2031 322.35 379.25 322.2563 Q379.3125 322.1625 379.4531 322.1 Q379.6094 322.0219 379.8594 321.9906 Q380.1094 321.9594 380.4844 321.9594 Q380.875 321.9594 381.125 321.9906 Q381.375 322.0219 381.5156 322.1 Q381.6562 322.1625 381.7188 322.2563 Q381.7812 322.35 381.7812 322.4594 L381.7812 325.8031 L385.3906 325.8031 Q385.5156 325.8031 385.6094 325.8656 Q385.7031 325.9125 385.7656 326.0531 Q385.8438 326.1781 385.875 326.3813 Q385.9062 326.5844 385.9062 326.8813 Q385.9062 327.4438 385.7656 327.6938 Q385.625 327.9438 385.3906 327.9438 L381.7812 327.9438 L381.7812 335.7875 Q381.7812 337.2406 382.2031 337.9906 Q382.6406 338.725 383.75 338.725 Q384.1094 338.725 384.3906 338.6625 Q384.6719 338.5844 384.8906 338.5063 Q385.1094 338.4281 385.2656 338.3656 Q385.4219 338.2875 385.5469 338.2875 Q385.625 338.2875 385.6875 338.3344 Q385.7656 338.3656 385.7969 338.475 Q385.8438 338.5844 385.875 338.7719 Q385.9062 338.9594 385.9062 339.2406 ZM391.8906 340.2719 Q391.8906 340.3969 391.8281 340.4906 Q391.7656 340.5688 391.625 340.6313 Q391.4844 340.6938 391.2344 340.725 Q390.9844 340.7563 390.5938 340.7563 Q390.2188 340.7563 389.9688 340.725 Q389.7188 340.6938 389.5625 340.6313 Q389.4219 340.5688 389.3594 340.4906 Q389.3125 340.3969 389.3125 340.2719 L389.3125 319.3969 Q389.3125 319.2719 389.3594 319.1781 Q389.4219 319.0844 389.5625 319.0219 Q389.7188 318.9594 389.9688 318.9281 Q390.2188 318.8969 390.5938 318.8969 Q390.9844 318.8969 391.2344 318.9281 Q391.4844 318.9594 391.625 319.0219 Q391.7656 319.0844 391.8281 319.1781 Q391.8906 319.2719 391.8906 319.3969 L391.8906 340.2719 ZM408.6562 332.6313 Q408.6562 333.2406 408.3438 333.5063 Q408.0469 333.7563 407.6562 333.7563 L398.4375 333.7563 Q398.4375 334.9281 398.6719 335.8656 Q398.9062 336.8031 399.4531 337.475 Q400 338.1469 400.875 338.5063 Q401.75 338.8656 403.0156 338.8656 Q404.0156 338.8656 404.7969 338.7094 Q405.5781 338.5375 406.1406 338.3344 Q406.7188 338.1313 407.0781 337.975 Q407.4531 337.8031 407.6406 337.8031 Q407.75 337.8031 407.8281 337.8656 Q407.9219 337.9125 407.9688 338.0219 Q408.0156 338.1313 408.0312 338.3344 Q408.0625 338.5219 408.0625 338.8031 Q408.0625 339.0063 408.0469 339.1625 Q408.0312 339.3031 408 339.4281 Q407.9844 339.5375 407.9219 339.6313 Q407.875 339.725 407.7812 339.8188 Q407.7031 339.8969 407.2812 340.1 Q406.8594 340.2875 406.1875 340.475 Q405.5156 340.6625 404.625 340.8031 Q403.75 340.9594 402.75 340.9594 Q401.0156 340.9594 399.7031 340.475 Q398.4062 339.9906 397.5156 339.0375 Q396.625 338.0844 396.1719 336.6469 Q395.7188 335.2094 395.7188 333.3031 Q395.7188 331.4906 396.1875 330.0531 Q396.6562 328.6 397.5312 327.6 Q398.4219 326.5844 399.6719 326.0531 Q400.9219 325.5063 402.4688 325.5063 Q404.125 325.5063 405.2812 326.0375 Q406.4531 326.5688 407.2031 327.475 Q407.9531 328.3656 408.2969 329.5844 Q408.6562 330.7875 408.6562 332.1625 L408.6562 332.6313 ZM406.0625 331.8656 Q406.1094 329.8344 405.1562 328.6781 Q404.2188 327.5219 402.3594 327.5219 Q401.4062 327.5219 400.6875 327.8813 Q399.9688 328.2406 399.4844 328.8344 Q399 329.4281 398.7344 330.225 Q398.4688 331.0063 398.4375 331.8656 L406.0625 331.8656 Z" style="stroke:none;"/>
    </g>
    <g style="color-interpolation:linearRGB; color-rendering:optimizeQuality; fill:rgb(255,255,255); fill-opacity:0; image-rendering:optimizeQuality; stroke:rgb(255,255,255); stroke-linecap:butt; stroke-linejoin:round; stroke-opacity:0; stroke-width:0.25; text-rendering:optimizeLegibility;">
      <path d="M108 306 L612 306 L612 444 L108 444 Z" style="fill:none;"/>
    </g>
  </g>
</svg>
